src/tests/functional/plugin/cpu/single_layer_tests/gru_sequence.cpp

```diff
@@ -105,15 +105,11 @@ class GRUSequenceCPUTest : public testing::WithParamInterface<GRUSequenceCpuSpec
         const size_t inputSize = targetStaticShapes.front()[0][2];
         const size_t numDirections = direction == ov::op::RecurrentSequenceDirection::BIDIRECTIONAL ? 2 : 1;
 
-        // 3rd input type must be an integer, thus it cannot be forced to BF16.
         if (additionalConfig[InferenceEngine::PluginConfigParams::KEY_ENFORCE_BF16] == InferenceEngine::PluginConfigParams::YES) {
-            if (inputDynamicShapes.size() > 2)
-                throw std::runtime_error("Invalid test case. Cannot enforce integer input to BF16.");
```

Comment:
Just wondering why this condition was deleted? Are we changing integer precision to bf16 now?

Reply:
There was approach which forces BF16 precision via inType to all inputs, thus there was needed such validation. Now BF16 is forced in the CPU plug-in.